src/vivarium_ciff_sam/constants/models.py

```diff
@@ -15,22 +15,20 @@ def __new__(cls, value):
 ###########################
 
 # TODO input details of model states and transitions
-# SOME_MODEL_NAME = data_keys.SOME_DISEASE.name
-# SUSCEPTIBLE_STATE_NAME = f'susceptible_to_{SOME_MODEL_NAME}'
-# FIRST_STATE_NAME = 'first_state'
-# SECOND_STATE_NAME = 'second_state'
-# SOME_DISEASE_MODEL_STATES = (SUSCEPTIBLE_STATE_NAME, FIRST_STATE_NAME, SECOND_STATE_NAME)
-# SOME_DISEASE_MODEL_TRANSITIONS = (
-#     TransitionString(f'{SUSCEPTIBLE_STATE_NAME}_TO_{FIRST_STATE_NAME}'),
-#     TransitionString(f'{FIRST_STATE_NAME}_TO_{SECOND_STATE_NAME}'),
-#     TransitionString(f'{SECOND_STATE_NAME}_TO_{FIRST_STATE_NAME}')
-# )
+DIARRHEA_MODEL_NAME = data_keys.DIARRHEA.name
```

Comment:
Since we're using the standard tools from VPH, we don't actually need these constants until results processing. I'll add the rest then